tests/docs/test_docs.py

```diff
@@ -11,16 +11,22 @@
 from ..conftest import _CHEM_INSTALLED
 from .utils import extract_code_blocks
 
-doc_files = ["README.md", *Path("docs/userguide/").rglob("*.md")]
+doc_files = [Path("README.md"), *Path("docs/userguide/").rglob("*.md")]
 """Files whose code blocks are to be checked."""
+
 doc_files_pseudocode = [Path("docs/userguide/campaigns.md")]
-"""Files which contain pseudocode that needs to be checked using fixtures."""
+"""Files containing pseudocode that needs to be checked with injected fixtures."""
```

Comment:
need

Reply:
nope, `needs` is indeed correct. refers to `pseudocode`, not `files` (that's also how it used to be prior to the change, btw)